actions/add_slb_server.yml

```diff
@@ -37,3 +37,6 @@ parameters:
           - True
           - False
         default: False
+    appliance:
+        required: true
+        default: default
```

Comment:
I think that this parameter should declare a type, for example: `type: string`.

I also think that it should not be required. I don't think it makes sense to have it `required: true` AND have a default value.